TabGroup/tests/TabGroup-specs.js

```diff
@@ -1,11 +1,30 @@
 import '@testing-library/jest-dom';
 import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import Button from '../../Button';
 import TabGroup from '../TabGroup';
 import ThemeDecorator from '../../ThemeDecorator';
 
 describe('TabGroup Specs', () => {
+	test('should fire `onTabClick` with `onTabClick` type when a tab is clicked', () => {
+		const handleTabClick = jest.fn();
+		render(
+			<TabGroup
+				tabPosition="before"
+				tabs={[
+					{title: 'Home', icon: 'home', onTabClick: handleTabClick},
```

Comment:
When I tried with "selected:true"

{title: 'Home', icon: 'home', selected:true}, It does not work neither.
Not all props we put here will be passed to TabBase. (I don't know whether it is a bug or original intention)

I guess the method of putting onClick here wasn't supported from the beginning. (Because TabGroup support onSelect)
I wonder if it is correct to fix about onClick.

Reply:
Whenever I tried this fix with onClick prop, inside GroupItem.js(@enact/ui) it's being overwritten into handleSelect this resulting into a bug because onClick is not being passed further to the component. 
![image](https://user-images.githubusercontent.com/111972354/210809759-8637493f-9cba-4b0a-95e8-1d48f56dcaca.png)
What do you suggest?

Reply:
> I guess the method of putting onClick here wasn't supported from the beginning. (Because TabGroup support onSelect)
I wonder if it is correct to fix about onClick.

I think the onClick included in each item may not have been intentionally disconnected.
Is there any problem if this is not supported?
Supporting onClick doesn't seem to have any documentation or examples.